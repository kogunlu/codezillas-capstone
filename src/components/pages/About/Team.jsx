import React from "react";
import TeamCard from "./TeamCard";
import team1 from "./team/team1.jpg"
import team2 from "./team/team2.jpg"
import team3 from "./team/team3.jpg"
import team4 from "./team/team4.jpg"
import team5 from "./team/team5.jpg"
import team6 from "./team/team6.jpg"

// About Page- Meet the Team. TeamCards.jsx called there
function Team() {
    return (
    <div className="pb-40">
        <div className="w-full h-full px-5 py-20 md:px-24 lg:pb-20 lg:px-32 xl:px-64 flex flex-col  gap-20 xl:gap-40">
        <h1 className="text-4xl font-normal 2xl:text-6xl 2xl:pt-24">WE ARE HEALING, NICE TO MEET YOU!</h1>
        <p className="text-2xl  text-gray-800 2xl:text-4xl">Meet the Team!</p>
        </div>

        <div className="flex flex-wrap justify-center">
            <TeamCard img={team1} name="Allan Saleh" content="Lead Engineer & Web Developer"/>
            <TeamCard img={team2} name="Payam Abubakr" content="Junior Designer & FE Developer"/>
            <TeamCard img={team3} name="Ahmed Mohammed" content="Junior Designer & FE Developer"/>
            <TeamCard img={team4} name="Abdulmajeed Jaafer" content="Junior Designer & FE Developer"/>
            <TeamCard img={team5} name="Snoor Madih" content="Junior Designer & FE Developer"/>
            <TeamCard img={team6} name="Avraz Zebary" content="Junior Designer & FE Developer"/>
        </div>


    </div>
    )
}

export default Team