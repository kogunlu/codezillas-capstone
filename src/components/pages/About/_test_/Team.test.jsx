import React from "react";
import renderer from "react-test-renderer";
import Team from "../Team";

it('Renders correctly for Team Component', () => {
    const tree = renderer
    .create(<Team />)
    .toJSON();
    expect(tree).toMatchSnapshot();
  });