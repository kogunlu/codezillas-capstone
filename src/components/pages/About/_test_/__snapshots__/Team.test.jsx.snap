// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly for Team Component 1`] = `
<div
  className="pb-40"
>
  <div
    className="w-full h-full px-5 py-20 md:px-24 lg:pb-20 lg:px-32 xl:px-64 flex flex-col  gap-20 xl:gap-40"
  >
    <h1
      className="text-4xl font-normal 2xl:text-6xl 2xl:pt-24"
    >
      WE ARE HEALING, NICE TO MEET YOU!
    </h1>
    <p
      className="text-2xl  text-gray-800 2xl:text-4xl"
    >
      Meet the Team!
    </p>
  </div>
  <div
    className="flex flex-wrap justify-center"
  >
    <div
      className="flex flex-col m-3 w-36  xl:w-36 2xl:w-80 hover:scale-105 hover:shadow-lg rounded-3xl"
    >
      <img
        alt="Team Member"
        className="object-contain"
        src="team1.jpg"
      />
      <div
        className="bg-cyan-50 rounded-b-3xl"
      >
        <p
          className="h-20 pt-8 text-center border-t-green-400 text-lg 2xl:text-3xl"
        >
          Allan Saleh
        </p>
        <p
          className="pt-8 pb-20 text-center text-base 2xl:text-2xl"
        >
          Lead Engineer & Web Developer
        </p>
      </div>
    </div>
    <div
      className="flex flex-col m-3 w-36  xl:w-36 2xl:w-80 hover:scale-105 hover:shadow-lg rounded-3xl"
    >
      <img
        alt="Team Member"
        className="object-contain"
        src="team2.jpg"
      />
      <div
        className="bg-cyan-50 rounded-b-3xl"
      >
        <p
          className="h-20 pt-8 text-center border-t-green-400 text-lg 2xl:text-3xl"
        >
          Payam Abubakr
        </p>
        <p
          className="pt-8 pb-20 text-center text-base 2xl:text-2xl"
        >
          Junior Designer & FE Developer
        </p>
      </div>
    </div>
    <div
      className="flex flex-col m-3 w-36  xl:w-36 2xl:w-80 hover:scale-105 hover:shadow-lg rounded-3xl"
    >
      <img
        alt="Team Member"
        className="object-contain"
        src="team3.jpg"
      />
      <div
        className="bg-cyan-50 rounded-b-3xl"
      >
        <p
          className="h-20 pt-8 text-center border-t-green-400 text-lg 2xl:text-3xl"
        >
          Ahmed Mohammed
        </p>
        <p
          className="pt-8 pb-20 text-center text-base 2xl:text-2xl"
        >
          Junior Designer & FE Developer
        </p>
      </div>
    </div>
    <div
      className="flex flex-col m-3 w-36  xl:w-36 2xl:w-80 hover:scale-105 hover:shadow-lg rounded-3xl"
    >
      <img
        alt="Team Member"
        className="object-contain"
        src="team4.jpg"
      />
      <div
        className="bg-cyan-50 rounded-b-3xl"
      >
        <p
          className="h-20 pt-8 text-center border-t-green-400 text-lg 2xl:text-3xl"
        >
          Abdulmajeed Jaafer
        </p>
        <p
          className="pt-8 pb-20 text-center text-base 2xl:text-2xl"
        >
          Junior Designer & FE Developer
        </p>
      </div>
    </div>
    <div
      className="flex flex-col m-3 w-36  xl:w-36 2xl:w-80 hover:scale-105 hover:shadow-lg rounded-3xl"
    >
      <img
        alt="Team Member"
        className="object-contain"
        src="team5.jpg"
      />
      <div
        className="bg-cyan-50 rounded-b-3xl"
      >
        <p
          className="h-20 pt-8 text-center border-t-green-400 text-lg 2xl:text-3xl"
        >
          Snoor Madih
        </p>
        <p
          className="pt-8 pb-20 text-center text-base 2xl:text-2xl"
        >
          Junior Designer & FE Developer
        </p>
      </div>
    </div>
    <div
      className="flex flex-col m-3 w-36  xl:w-36 2xl:w-80 hover:scale-105 hover:shadow-lg rounded-3xl"
    >
      <img
        alt="Team Member"
        className="object-contain"
        src="team6.jpg"
      />
      <div
        className="bg-cyan-50 rounded-b-3xl"
      >
        <p
          className="h-20 pt-8 text-center border-t-green-400 text-lg 2xl:text-3xl"
        >
          Avraz Zebary
        </p>
        <p
          className="pt-8 pb-20 text-center text-base 2xl:text-2xl"
        >
          Junior Designer & FE Developer
        </p>
      </div>
    </div>
  </div>
</div>
`;
